libs/ui/src/components/atoms/Dropdown/Dropdown.tsx

```diff
@@ -90,6 +86,6 @@ const DropdownContentFactory = ({
         }
         return null;
       })}
-    </DropdownMenuContent>
+    </>
```

Comment:
just noticed this one warning on the fragment.

Reply:
ty